packages/components/src/components/Heading/Heading.tsx

```diff
@@ -29,25 +28,15 @@ type HeadingSizeBreakpoints = {
 
 type HeadingSizeOptionsProp = HeadingSizeBreakpoints | HeadingSizeOptions;
 
-type HeadingFontColors =
-  | "colorTextHeading"
-  | "colorTextHeadingStrong"
-  | "colorTextHeadingStronger"
-  | "colorTextInverse"
-  | "colorTextStrongest";
-
-export interface HeadingProps extends React.HTMLAttributes<HTMLHeadingElement> {
+export interface HeadingProps
+  extends Omit<React.HTMLAttributes<HTMLHeadingElement>, "color">,
+    BoxProps {
   /** Sets the HTML element on render. */
   as?: HeadingLevelOptions;
   /** The contents of the heading. Can be text or valid text related HTML, i.e. anchor and strong elements. */
   children: NonNullable<React.ReactNode>;
-  /** Changes the bottom margin of the heading.  */
-  marginBottom?: HeadingMarginOptions;
   /** Changes the size of the heading. */
   size?: HeadingSizeOptionsProp;
```

Comment:
If we pass the prop `color` here as BoxProps["color"], we don't need to cast it in here https://github.com/Localitos/pluto/pull/1675/files?diff=split&w=1#diff-79d1fad6284384507b904515127be29ac7ca2e1a6d2ffb387c1fc8af0c9fa7e1R149.

Reply:
I got an error while doing it:

```
The expected type comes from property 'color' which is declared here on type 'IntrinsicAttributes & { col?: SystemProp<string | number | true, Theme> | undefined; p?: PaddingProp<Theme> | undefined; ... 440 more ...; listStylePosition?: SystemProp<...> | undefined; } & { ...; } & { ...; }'
```